templates/openwisp2/urls.py

```diff
@@ -12,7 +12,7 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    {% if openwisp2_controller_urls %}
+    {% if openwisp2_controller_urls or openwisp2_monitoring %}
```

Comment:
why this addition?

Reply:
The [README](https://github.com/openwisp/openwisp-monitoring#setup-integrate-in-an-existing-django-project) of openwisp-monitoring suggests adding `openiwsp-controller`'s urlpatterns. Should I remove this? @nemesisdesign

Reply:
I would remove this change and follow the structure of the current code and add a new var called `openwisp2_monitoring_urls` which allows disabling the URLs if needed.
In some cases the application has to be installed on the VM/container but the URLs may be disabled because are served by another VM/container.